### PR TITLE
Update add channel members graph documentation to display maximum bul…

### PR DESCRIPTION
**Pull request by kallu800**
Update add channel members graph documentation to display maximum bulk membership size

### CODE REVIEW

**Review by akhilkohlimicrosoft — APPROVED** [review]

File: api-reference/beta/resources/channel.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ where files are shared, and where tabs are added.
 |[Get files folder](../api/channel-get-filesfolder.md)| [driveItem](driveitem.md) | Retrieves the details of the SharePoint folder where the files for the channel are stored. |
 |[List tabs](../api/channel-list-tabs.md) | [teamsTab](teamstab.md) | Lists tabs pinned to a channel.|
 |[List channel members](../api/channel-list-members.md) | [conversationMember](conversationmember.md) collection | Get the list of members in a channel.|
-|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for channels with a **membershipType** of `private` or `shared`.|
+|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for channels with a **membershipType** of `private` or `shared`. A maximum of 200 members can be added per request.
```
Comment (shantanu1singh) [inline api-reference/beta/resources/channel.md:38]: I don't think we should include the limit in this list. The primary API reference doc is sufficient.

File: api-reference/v1.0/resources/channel.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ where files are shared, and where tabs are added.
 |[Get files folder](../api/channel-get-filesfolder.md)| [driveItem](driveitem.md) | Retrieves the details of the SharePoint folder where the files for the channel are stored. |
 |[List tabs](../api/channel-list-tabs.md) | [teamsTab](teamstab.md) | Lists tabs pinned to a channel.|
 |[List channel members](../api/channel-list-members.md) | [conversationMember](conversationmember.md) collection | Get the list of members in a channel.|
-|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for `channel` with membershipType of `private`.|
+|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for `channel` with membershipType of `private`. A maximum of 200 members can be added per request.
```
Comment (shantanu1singh) [inline api-reference/v1.0/resources/channel.md:34]: I don't think we should include the limit in this list. The primary API reference doc is sufficient.

File: concepts/known-issues.md [hunk 3]
```diff
@@ -398,6 +398,9 @@ Role query filters along with other filters `GET /teams/team-id/members?$filter=
 
 All the requests to filter team members by roles expect either a _skipToken_ parameter or a _top_ paramater in the request, but not both. If both the parameters are passed in the request, the _top_ parameter will be ignored.
 
+### Unable to add 200+ members per request to a channel
+A maximum of 200 members per request can be added to a channel.
```
Comment (shantanu1singh) [inline concepts/known-issues.md:402]: Let's specify that it's only for private channels and give a link to the main API reference doc here as well.

Comment (shantanu1singh) [reply concepts/known-issues.md:402]: Main/primary API reference doc being 'channel-post-members.md'

File: api-reference/v1.0/api/channel-post-members.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ ms.prod: "microsoft-teams"
 
 Namespace: microsoft.graph
 
-Add a [conversationMember](../resources/conversationmember.md) to a [channel](../resources/channel.md). This operation is allowed only for channels with a **membershipType** value of `private`.
+Add a [conversationMember](../resources/conversationmember.md) to a [channel](../resources/channel.md). This operation is allowed only for channels with a **membershipType** value of `private`. A maximum of 200 members can be added per request.
```
Comment (shantanu1singh) [inline api-reference/v1.0/api/channel-post-members.md:14]: Is the 200 limit only per request or private channels in *total* can only support 200 members?

**Comment by Lauragra** [comment]
@kallu800 , please address the review feedback before you set the **ready to merge** label.

File: api-reference/beta/resources/channel.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ where files are shared, and where tabs are added.
 |[Get files folder](../api/channel-get-filesfolder.md)| [driveItem](driveitem.md) | Retrieves the details of the SharePoint folder where the files for the channel are stored. |
 |[List tabs](../api/channel-list-tabs.md) | [teamsTab](teamstab.md) | Lists tabs pinned to a channel.|
 |[List channel members](../api/channel-list-members.md) | [conversationMember](conversationmember.md) collection | Get the list of members in a channel.|
-|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for channels with a **membershipType** of `private` or `shared`.|
+|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for channels with a **membershipType** of `private` or `shared`.
```
Comment (shantanu1singh) [inline api-reference/beta/resources/channel.md:38]: Please restore the closing |  at the end.

File: api-reference/v1.0/resources/channel.md [hunk 6]
```diff
@@ -31,7 +31,7 @@ where files are shared, and where tabs are added.
 |[Get files folder](../api/channel-get-filesfolder.md)| [driveItem](driveitem.md) | Retrieves the details of the SharePoint folder where the files for the channel are stored. |
 |[List tabs](../api/channel-list-tabs.md) | [teamsTab](teamstab.md) | Lists tabs pinned to a channel.|
 |[List channel members](../api/channel-list-members.md) | [conversationMember](conversationmember.md) collection | Get the list of members in a channel.|
-|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for `channel` with membershipType of `private`.|
+|[Add channel member](../api/channel-post-members.md) | [conversationMember](conversationmember.md) | Add a member to a channel. Only supported for `channel` with membershipType of `private`.
```
Comment (shantanu1singh) [inline api-reference/v1.0/resources/channel.md:34]: Please restore the closing |  at the end.

File: api-reference/beta/api/channel-post-members.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Add a [conversationMember](../resources/conversationmember.md) to a [channel](../resources/channel.md). This operation is allowed only for channels with a **membershipType** value of `private` or `shared`.
+Add a [conversationMember](../resources/conversationmember.md) to a [channel](../resources/channel.md). This operation is allowed only for channels with a **membershipType** value of `private` or `shared`. A maximum of 200 members can be added per request.
```
Comment (shantanu1singh) [inline api-reference/beta/api/channel-post-members.md:16]: Is the limit only for private channels or shared channels too? Please clarify that here

File: concepts/known-issues.md [hunk 8]
```diff
@@ -398,6 +398,9 @@ Role query filters along with other filters `GET /teams/team-id/members?$filter=
 
 All the requests to filter team members by roles expect either a _skipToken_ parameter or a _top_ paramater in the request, but not both. If both the parameters are passed in the request, the _top_ parameter will be ignored.
 
+### Unable to add 200+ members per request to a private channel
+A maximum of 200 members per request can be added to a private channel. Here is the link to the api reference doc : (/microsoft-graph-docs/api-reference/v1.0/api/channel-post-members.md) and (/microsoft-graph-docs/api-reference/beta/api/channel-post-members.md)
```
Comment (shantanu1singh) [inline concepts/known-issues.md:402]: would recommend rewording to '...added to a private channel through the `POST /teams/{team-id}/channels/{channel-id}/members` request.'

I'll let @Lauragra or @isvargasmsft recommend the right framing.

Comment (Lauragra) [reply concepts/known-issues.md:402]: Revise the second sentence as:
For details, see [Add member to channel](/microsoft-graph-docs/api-reference/v1.0/api/channel-post-members).

**Review by akhilkohlimicrosoft — CHANGES_REQUESTED** [review]

File: concepts/known-issues.md [hunk 9]
```diff
@@ -398,6 +398,9 @@ Role query filters along with other filters `GET /teams/team-id/members?$filter=
 
 All the requests to filter team members by roles expect either a _skipToken_ parameter or a _top_ paramater in the request, but not both. If both the parameters are passed in the request, the _top_ parameter will be ignored.
 
+### Unable to add 200+ members while creating a private channel
+A maximum of 200 members can be added while creating a private channel through the POST /teams/{team-id}/channels request.. For details, see [Create Channel](channel-post.md).
```
Comment (shantanu1singh) [inline concepts/known-issues.md:402]: Extra period after request

Comment (shantanu1singh) [inline concepts/known-issues.md:402]: enclose POST /teams/{team-id}/channels between ``.

`POST /teams/{team-id}/channels`

File: concepts/known-issues.md [hunk 10]
```diff
@@ -398,6 +398,9 @@ Role query filters along with other filters `GET /teams/team-id/members?$filter=
 
 All the requests to filter team members by roles expect either a _skipToken_ parameter or a _top_ paramater in the request, but not both. If both the parameters are passed in the request, the _top_ parameter will be ignored.
 
+### Unable to add 200+ members while creating a private channel
```
Comment (shantanu1singh) [inline concepts/known-issues.md:401]: maybe rephrase to 'Unable to add more than 200 members while creating a private channel' ?

File: api-reference/beta/api/channel-post.md [hunk 11]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Create a new [channel](../resources/channel.md) in a team, as specified in the request body.
+Create a new [channel](../resources/channel.md) in a team, as specified in the request body. A maximum of 200 members can be added while creating a private channel.
```
Comment (shantanu1singh) [inline api-reference/beta/api/channel-post.md:16]: Can we add this as a callout Note instead. Example:

> **Note:** The `$top` OData query parameter has a limit of 1000 alerts. We recommend that you include only `$top` and not `$skip` in your first GET query. You can use `@odata.nextLink` for pagination. If you need to use `$skip`, it has a limit of 500 alerts. For example, `/security/alerts?$top=10&$skip=500` will return a `200 OK` response code, but `/security/alerts?$top=10&$skip=501` will return a `400 Bad Request` response code. For more information, see [Microsoft Graph Security API error responses](../resources/security-error-codes.md).

Comment (shantanu1singh) [reply api-reference/beta/api/channel-post.md:16]: You can refer to this on how to add that: https://github.com/microsoftgraph/microsoft-graph-docs/blob/d96f678e36cd81439bb2d19214c55233a2a5495e/api-reference/beta/api/alert-list.md

**Review by shantanu1singh — CHANGES_REQUESTED** [review]

**Review by akhilkohlimicrosoft — APPROVED** [review]

File: concepts/known-issues.md [hunk 12]
```diff
@@ -398,6 +398,9 @@ Role query filters along with other filters `GET /teams/team-id/members?$filter=
 
 All the requests to filter team members by roles expect either a _skipToken_ parameter or a _top_ paramater in the request, but not both. If both the parameters are passed in the request, the _top_ parameter will be ignored.
 
+### Unable to add more than 200 members while creating a private channel
+A maximum of 200 members can be added while creating a private channel through the "POST /teams/{team-id}/channels" request. For details, see [Create Channel](channel-post.md).
```
Comment (shantanu1singh) [inline concepts/known-issues.md:402]: Instead of the " character, use backticks ` to enclose POST /teams/...channels. That way you'll get the correct formatting

**Review by shantanu1singh — APPROVED** [review]
Left one minor comment. LGTM otherwise.

@Lauragra @isvargasmsft can either of you help review too?

**Review by akhilkohlimicrosoft — APPROVED** [review]

File: api-reference/beta/api/channel-post.md [hunk 13]
```diff
@@ -14,6 +14,7 @@ Namespace: microsoft.graph
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
 Create a new [channel](../resources/channel.md) in a team, as specified in the request body.
+> **Note:** A maximum of 200 members can be added while creating a private channel.
```
Comment (Lauragra) [inline api-reference/beta/api/channel-post.md:17]: Suggested revision to the note text: "A private channel can have a maximum of 200 members."

Does this API also create channels that aren't private, and if so, is there a maximum number of members?

File: concepts/known-issues.md [hunk 14]
```diff
@@ -398,6 +398,9 @@ Role query filters along with other filters `GET /teams/team-id/members?$filter=
 
 All the requests to filter team members by roles expect either a _skipToken_ parameter or a _top_ paramater in the request, but not both. If both the parameters are passed in the request, the _top_ parameter will be ignored.
 
+### Unable to add more than 200 members while creating a private channel
```
Comment (Lauragra) [inline concepts/known-issues.md:401]: Suggested revision to the heading to be more clear/concise: Unable to add more than 200 members to a private channel

File: api-reference/beta/api/channel-post.md (hunk 13 above)
Comment (kallu800) [reply api-reference/beta/api/channel-post.md:17]: So, at the creation of a private channel, you can add only upto 200 members in a request. A private channel can have a max of 200 members would be wrong as members can be added later using the AddMember api.

We checked this limit only for private channel.

File: concepts/known-issues.md [hunk 15]
```diff
@@ -398,6 +398,9 @@ Role query filters along with other filters `GET /teams/team-id/members?$filter=
...
+A maximum of 200 members can be added while creating a private channel through the `POST /teams/{team-id}/channels` request. For details, see [Create Channel](channel-post.md).
```
[the remark this answers is not in the document]
Comment (kallu800) [reply concepts/known-issues.md:402]: Sure, I will remove my changes from this file.

File: api-reference/beta/api/channel-post.md (hunk 13 above)
Comment (Lauragra) [reply api-reference/beta/api/channel-post.md:17]: Okay, suggested revision: "You can add a maximum of 200 members when you create a private channel."

**Review by shantanu1singh — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]
